### PR TITLE
Update README info

### PR DESCRIPTION
**Pull request by uilianries**
I just executed conan readme generator to update README info.

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -53,9 +63,10 @@ def build_configure(self):
             else:
                 args.extend(['--disable-shared', '--enable-static'])
             env_build = AutoToolsBuildEnvironment(self)
-            env_build.configure(args=args)
-            env_build.make()
-            env_build.make(args=['install'])
+            with tools.environment_append(env_build.vars):
+                self._run_cmd("./configure %s" % ' '.join(args))
+                self._run_cmd("make")
```
Comment (SSE4) [inline conanfile.py]: this needs -j % tools.cpu_count()

**Comment by SSE4** [comment]
MinGW build timed out, please split builds (e.g. x86 and x64) or use pages

**Comment by uilianries** [comment]
I'm going to change now

**Comment by uilianries** [comment]
@SSE4 Should we insist to solve MSVC 12? Regarding from other recipes, this version is a pain.

**Comment by SSE4** [comment]
@uilianries if library itself supports VS12, I think we should solve
otherwise, we have to add checks to prevent such builds

**Comment by SSE4** [comment]
@uilianries please rebase

File: conanfile.py [hunk 2]
```diff
@@ -9,10 +9,11 @@ class LibsodiumConan(ConanFile):
     name = "libsodium"
     version = "1.0.15"
     url = "https://github.com/bincrafters/conan-libsodium"
+    homepage = "https://github.com/jedisct1/libsodium"
     description = "Sodium is a modern, easy-to-use software library for encryption, decryption, signatures, " \
                   "password hashing and more."
     license = "https://github.com/jedisct1/libsodium/blob/master/LICENSE"
-    exports_sources = ["LICENSE", "FindSodium.cmake"]
```
Comment (SSE4) [inline conanfile.py]: this shouldn't be removed

**Review by SSE4 — CHANGES_REQUESTED** [review]

File: conanfile.py (hunk 2 above)
Comment (uilianries) [reply conanfile.py]: Some messy occurred during my rebase, thanks for review

Comment (uilianries) [reply conanfile.py]: done!

**Comment by uilianries** [comment]
BTW, there is a new patch version 1.0.16

After to finish this PR and merge to stable, I could bump the new version.

**Review by SSE4 — APPROVED** [review]

**Comment by SSE4** [comment]
@uilianries great, go ahead!

**Comment by SSE4** [comment]
merging as CI passed

**Pull request merged**